DBSCE/LocalKeyHelper-Android.md

```diff
@@ -0,0 +1,89 @@
+<span style="color:red">!!! DRAFT - Please note this page is still in draft as we work on platform specific details
+</span>
+
+### Local Key Helper on Android
```

Comment:
I'll not review this for now, since there're some open questions on implementation on the main PR

Reply:
Yes. Added a DRAFT notice. If it is better to remove them, and just say TBD, I can do that too before sharing this widely.